device-types/Cisco/C9800-40-K9.yaml

```diff
@@ -14,21 +15,24 @@ interfaces:
   - name: Management Port
     type: 1000base-t
     mgmt_only: true
-  - name: Redundancy Port
+  - name: Redundancy Ethernet Port
     type: 1000base-t
     mgmt_only: true
+  - name: Redundancy SFP Port
+    type: 1000base-x-sfp
+    mgmt_only: true
   - name: TenGigabitEthernet0/0/0
-    type: 10gbase-t
+    type: 10gbase-x-sfpp
   - name: TenGigabitEthernet0/0/1
-    type: 10gbase-t
+    type: 10gbase-x-sfpp
   - name: TenGigabitEthernet0/0/2
-    type: 10gbase-t
+    type: 10gbase-x-sfpp
   - name: TenGigabitEthernet0/0/3
-    type: 10gbase-t
-power-ports:
-  - name: PS1
-    type: iec-60320-c14
-    maximum_draw: 381
-  - name: PS2
-    type: iec-60320-c14
-    maximum_draw: 381
+    type: 10gbase-x-sfpp
+module-bays:
+  - name: PEM 0 Power Supply
```

Comment:
The names of the PSU module bays should match exactly what either A) the config shows, or B) the documentation shows

The documentation says `PEM 0`, so unless you can get a sample from the configuration lets make them `PEM 0`. If you can get a config output please paste it here so we have it for the future